media/css/firefox/developer/developer.scss

```diff
@@ -87,7 +87,8 @@
     // Hide it for now pending a better solution.
     #global-nav-download-firefox,
     #sub-nav-download-firefox,
-    #nav-download-firefox {
+    #nav-download-firefox,
```

Comment:
There's a [handy include here](https://github.com/mozilla/bedrock/blob/master/media/css/hubs/_nav-download-button-remove.scss) which means we don't have to rewrite this for every page.

Reply:
Nice... I suspect that didn't exist when I first built this page. I'll update.